gradle.properties

```diff
@@ -87,6 +87,7 @@ org.gradle.parallel = true
 buildCacheRetentionDays = 7
 
 kotlin.code.style = official
+kotlin.stdlib.default.dependency=false
```

Comment:
I think this is worth a code comment.

Reply:
I was hoping "Git blame" would be enough, but I can repeat the info from there in a code comment if you want.

Reply:
Yes, maybe git blame is enough.

Reply:
I had actually added a comment but forgot to push. See https://github.com/oss-review-toolkit/ort/pull/4974.